hub/util/keys.py

```diff
@@ -48,3 +52,13 @@ def tensor_exists(key: str, storage: StorageProvider) -> bool:
         return True
     except KeyError:
         return False
+    return get_tensor_meta_key(key) in storage
```

Comment:
remove line 55 and 64. They should never be reached. If it's found in try it'll return True, otherwise it'll be a keyeroor and it'll return false in the except.